src/client/coll/mod.rs

```diff
@@ -171,7 +171,7 @@ impl<'a> Collection<'a> {
 
     // Helper method for all findAndModify commands.
     fn find_and_modify(&self, cmd: &mut bson::Document,
-                       filter: bson::Document, max_time_ms: Option<i64>,
+                       filter: bson::Document, _max_time_ms: Option<i64>,
```

Comment:
Didn't meant to leave this in, I just did this to demonstrate to Kevin how to get rid of the `unused variable` warning. I'll put this in a separate pull request with other refactors.